Presentation.Web/Controllers/API/V1/AuthorizeController.cs

```diff
@@ -9,22 +9,27 @@
 using Presentation.Web.Infrastructure;
 using Presentation.Web.Models;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Net;
 using System.Net.Http.Headers;
 using System.Text;
 using System.Web;
 using System.Web.Helpers;
+using AutoMapper.Execution;
+using Castle.DynamicProxy.Generators.Emitters.SimpleAST;
```

Comment:
remember to cleanup unused usings. A lot of new usings here that we should not use